### PR TITLE
Centos8

### PR DESCRIPTION
**Pull request by jburel**
This PR adds installation instructions for CentOS 8

* Check that travis is green

Note that an URL fix for pg on CenOS 7 is also included this PR, a version bump happened today breaking the build

### CODE REVIEW

File: linux/step01_centos8_pg_deps.sh [hunk 1]
```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+
+PGVER=${PGVER:-pg11}
+
+
+#start-postgresql-installation-general
+#required to install postgresql, rpm Db corrupted
+rm -f /var/lib/rpm/__db*
```
Comment (joshmoore) [inline linux/step01_centos8_pg_deps.sh:7]: I'm concerned about having this in the final docs. Do we know that this won't match any other (non-PG) directories?

Comment (manics) [reply linux/step01_centos8_pg_deps.sh:7]: This is the main RPM db. It's possible to rebuild it if it becomes corrupted: https://serverfault.com/questions/94367/centos-rpm-database-trashed-rpm-rebuilddb-wont-fix-can-i-recover-using-v

But if our instructions lead to that corruption (i.e. it's not already corrupted in the base docker/VM image and it's not a yum/dnf bug) I don't think we should publish them.

Comment (jburel) [reply linux/step01_centos8_pg_deps.sh:7]: I moved it one step down to help during the test so it appears in the doc during that phase
i will revert that commit

Comment (jburel) [reply linux/step01_centos8_pg_deps.sh:7]: the command ``rm ...`` should no longer appear in the doc now

**Comment by jburel** [comment]
@joshmoore @manics ok to merge so I can fix #237 (certificates config also applies to c8 and I will rename file)? I rolled back the commit helping during review.

**Comment by jburel** [comment]
https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-docs-superbuild-build/68/ available

**Comment by joshmoore** [comment]
WFM

**Pull request merged**